src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_BUY = "buy";
+    private static final String OPERATION_SUPPLY = "supply";
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static String createReport(String[] dataArray) {
+        int amountBuy = 0;
+        int amountSupply = 0;
+        for (int i = 0; i <= dataArray.length - 2; i += 2) {
+            if (OPERATION_BUY.equals(dataArray[i])) {
+                amountBuy += Integer.parseInt(dataArray[i + 1]);
+            } else {
+                amountSupply += Integer.parseInt(dataArray[i + 1]);
+            }
+        }
+        return OPERATION_SUPPLY + "," + amountSupply
+                + System.lineSeparator() + OPERATION_BUY + ","
+                + amountBuy + System.lineSeparator()
+                + "result" + "," + (amountSupply - amountBuy);
+    }
 
+    private void writeToFile(String toFileName, String report) {
```

Comment:
private methods should be listed after public

Reply:
what about this? https://github.com/mate-academy/jv-work-with-file/pull/150#discussion_r691144004